### PR TITLE
Allow Content editing of HomePage

### PR DESCRIPTION
**Pull request by kave**
## Additions

- Allow editing of Latest Updates Section & Text below Hero

## Testing

- Create a `HomePage` and add content 

## Review

- @kurtw 
- @richaagarwal 
- @jimmynotjim 

## Screenshots

screen shot 2016-03-09 at 1 37 51 pm
screen shot 2016-03-09 at 1 37 42 pm

## Checklist

* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [x] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

**Comment by kave** [comment]
@jimmynotjim I noticed the hyperlinks seem to be in a different color when using the half link blob molecules. Are we missing a hyperlink css class or something in the template?

File: cfgov/jinja2/v1/_layouts/base.html [hunk 1]
```diff
@@ -213,7 +213,7 @@
 
     {# TODO: Remove Else conditional when this is fully moved to wagtail. #}
     {# Check and include page-level JavaScript. #}
-    {% if page and page.media %}
+    {% if page %}
```
Comment (richaagarwal) [inline cfgov/jinja2/v1/_layouts/base.html]: If there is a page but no page.media, no js will be included - is this intended or is that a situation we no longer anticipate?

Comment (sebworks) [inline cfgov/jinja2/v1/_layouts/base.html]: Does every page have a page object?

**Comment by schaferjh** [comment]
Question about the category options -- will the content editors be able to use all the categories under newsroom? This is an *extra* so feel free to say no for this initial version, but I had a query from EA indicating that they would like to be able to select "press release" "speech" etc (which are currently categories on the newsroom). 

If we did this, we would still want to be able to use main categories like "Blog" "Event" "Report" -- so if it's too finicky to enable both levels of categories right now, that's fine.

File: cfgov/jinja2/v1/_layouts/base.html (hunk 1 above)
Comment (kurtw) [inline cfgov/jinja2/v1/_layouts/base.html]: The `page` is the page object.

Comment (sebworks) [inline cfgov/jinja2/v1/_layouts/base.html]: No, Im just wondering if we can get rid of the if statement and just have the for loop?

Comment (kurtw) [inline cfgov/jinja2/v1/_layouts/base.html]: Maybe I confused your question. Every time a page is served via wagtail, it will have a `page` object.

**Comment by kurtw** [comment]
This looks good to me :+1:

File: cfgov/jinja2/v1/_layouts/base.html (hunk 1 above)
Comment (sebworks) [inline cfgov/jinja2/v1/_layouts/base.html]: The code is in `base.html` so it will run on non wagtail pages as well.

Comment (kurtw) [inline cfgov/jinja2/v1/_layouts/base.html]: Oh I see. I don't believe they do. Context is defined [here](https://github.com/cfpb/cfgov-refresh/blob/flapjack/cfgov/sheerlike/__init__.py#L90) and [here](https://github.com/cfpb/cfgov-refresh/blob/flapjack/cfgov/v1/__init__.py#L23) so as long as they don't include a `page` context processor or variable then we should be okay to have this conditional.

Comment (sebworks) [inline cfgov/jinja2/v1/_layouts/base.html]: cool, I would just test this on a Browse page because that is a Wagtail page with no `page.media` object.

Comment (kave) [inline cfgov/jinja2/v1/_layouts/base.html]: @richaagarwal ah yes, your right. i'll add it back in

**Comment by kave** [comment]
@schaferjh do they all have different icons?
screen shot 2016-03-10 at 10 31 30 am

**Pull request merged**